crates/ibc/Cargo.toml

```diff
@@ -27,7 +44,7 @@ mocks = ["tendermint-testgen", "clock", "std"]
 
 [dependencies]
 # Proto definitions for all IBC-related interfaces, e.g., connections or channels.
-ibc-proto = { version = "0.22.0", default-features = false }
+ibc-proto = { version = "0.22.0", default-features = false, git = "https://github.com/octopus-network/ibc-proto-rs.git", branch = "refactor-error-system" }
```

Comment:
let's remember not to merge while we have this

Reply:
done